docs/manifest.json

```diff
@@ -875,6 +875,18 @@
 		"markdown_source": "../packages/components/src/flex/flex/README.md",
 		"parent": "components"
 	},
+	{
+		"title": "FlyoutContent",
+		"slug": "flyout-content",
+		"markdown_source": "../packages/components/src/flyout/flyout-content/README.md",
+		"parent": "components"
+	},
```

Comment:
I think this should be removed if we're not exporting it right?

Reply:
I believe that in order to remove these lines, we should also delete `/packages/components/src/flyout/flyout-content/README.md` — should I go ahead and delete that file too?

Reply:
Wow, that's pretty wild.

My concern is that we're adding external documentation for an internal component. I guess we should remove the README? 🤷

Reply:
I tried removing only the lines in `docs/manifest.json` in https://github.com/WordPress/gutenberg/pull/32197/commits/aa02f1e13033bd5627a1bad00b11cec84ed14d9e — let's see if the CI tasks still pass, otherwise I will also delete the README

Reply:
Yep, the Static Analysis task is failing.

<img width="799" alt="Screenshot 2021-07-02 at 19 58 38" src="https://user-images.githubusercontent.com/1083581/124312631-ed0f7c80-db6f-11eb-8bd2-4ce6b6812a2b.png">


I'll go ahead and remove the README file